PROTO_tests/tests/numeric_test.py

```diff
@@ -206,13 +205,242 @@ def test_sin(self, num_type):
 
     @pytest.mark.parametrize("num_type", NO_BOOL)
     def test_cos(self, num_type):
-        na = np.linspace(1, 10, 10).astype(num_type)
+        if num_type == ak.float64:
+            na = np.linspace(0, 10)
+        else:
+            na = np.arange(0, 10).astype(num_type)
         pda = ak.array(na, dtype=num_type)
 
         assert np.allclose(np.cos(na), ak.cos(pda).to_ndarray())
         with pytest.raises(TypeError):
             ak.cos(np.array([range(0, 10)]).astype(num_type))
 
+    @pytest.mark.parametrize("num_type", NO_BOOL)
+    def test_tan(self, num_type):
+        if num_type == ak.float64:
+            na = np.linspace(0, 10)
+        else:
+            na = np.arange(0, 10).astype(num_type)
+        pda = ak.array(na, dtype=num_type)
+
+        assert np.allclose(np.tan(na), ak.tan(pda).to_ndarray())
```

Comment:
I feel like we should update all the `np.allclose` to have `equal_nan=True`. This doesn't seem to factor in for these examples, but would if we add the edge cases at some point

Reply:
Updated

Reply:
we still need to update this one to use `equal_nan=True`